src/shared/multiline-strings.ts

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Maybe `dedent.ts` would be a more obvious name for the module? It depends on whether we have related multi-line string formatting utilities that we want to add in future.

Reply:
I named it like this mainly for two reasons:

1. We could need another place to add functions in future, or we might want to expose some of the other currently module-scoped functions.
2. The only exposed function currently does a number of things which makes naming the module a bit harder.

I think we can probably name the module `trim-and-dedent.ts`, after the main exposed function.

If we need to expose `trimExtraLines` or `dedent`, it will still make sense, and adding more functions is something hard to foresee at this point.